config/analytics_pii.yml

```diff
@@ -43,3 +43,5 @@ shared:
   - given_name
   - family_name
   - email
+  :international_relocation_payments_eligibilities:
+  - passport_number
```

Comment:
not 100% but looking at https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2915 it suggests these need to go into the blocklist. Maybe double check with @slorek if you've not already